airbyte-integrations/connectors/source-postgres/src/test/java/io/airbyte/integrations/source/postgres/PostgresSourceTest.java

```diff
@@ -253,6 +262,45 @@ public void testCanReadUtf8() throws Exception {
     }
   }
 
+  @Test
+  public void testValuesChangedOnChangedSourceSchema() throws Exception {
```

Comment:
> I'm not sure I understand what's being tested in this test: IIUC this test is verifying that if the underlying schema is changed, the records read will be different
> 
> But won't this always be true? Across two different `reads`, even with without the changes in this PR the read will always return different records

updated this test with 4 different reads with the same catalog:
1 read - initial sync (integers)
2 read - nothing changed ==> values are same as in 1 read (integers)
3 read - added 1 record into the source table ==> added 1 airbyte record (integers)
4 read - user changed column definition from int to float ==> values in airbyte records become double

This test fully emulate behaviour from this [issue](https://github.com/airbytehq/airbyte/issues/21413)

Reply:
But you aren't actually checking for the actual change you're making here, correct? i.e. you aren't verifying the correct logs are being outputted. 

In that case, I think you should remove this test altogether since that change is a logging one (and not a validation)

Reply:
> But you aren't actually checking for the actual change you're making here, correct? i.e. you aren't verifying the correct logs are being outputted.
> 
> In that case, I think you should remove this test altogether since that change is a logging one (and not a validation)

removed the test